frontend/shared/components/src/filters/components/GroupUIFilterList.vue

```diff
@@ -7,7 +7,13 @@
                 </template>
 
                 <div v-if="!isGroup(child)">
-                    <span v-for="(s, i) in child.styledDescription" :key="i" :class="'styled-description ' + s.style" v-text="s.text" />
+                    <template v-for="(s, i) in child.styledDescription" :key="i">
+                        <span v-if="s.choices?.length" class="button text inline margin-space" :class="s.style" type="button" @click.stop.prevent="$event => showChoices($event, s.choices!)">
```

Comment:
Denk dat het beter is om de margin-space classe in GroupUIFilterList te plaatsen in plaats van in buttons.scss, het is zo specifiek dat het nergens anders gebruikt kan worden